server/channels/app/scheduled_post_job.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 const (
-	getPendingScheduledPostsPageSize = 10
+	getPendingScheduledPostsPageSize = 100
```

Comment:
given the nature of the feature I wonder if it is possible to give these values the option to be rewritten via env variable/config? thoughts?

Reply:
We don't use batch size in config in any place as far as I can see. To be honest there is no perfect value for this. I'll tweak this during performance testing so that should be fine.